nixos/modules/services/databases/lldap.nix

```diff
@@ -107,8 +107,17 @@ in
       wants = [ "network-online.target" ];
       after = [ "network-online.target" ];
       wantedBy = [ "multi-user.target" ];
+      script = lib.optionalString (!cfg.settings ? jwt_secret) ''
+        if [[ -z "$LLDAP_JWT_SECRET_FILE" ]] && [[ -z "$LLDAP_JWT_SECRET" ]]; then
```

Comment:
This should have a comment explaining the Problem.

Reply:
Yes, good idea.
Pushed a comment a few moments ago :)